pkg/blackduck/ctl_blackduck.go

```diff
@@ -480,25 +483,26 @@ func (ctl *HelmValuesFromCobraFlags) GenerateHelmFlagsFromCobraFlags(flagset *pf
 					return
 				}
 				securityContextIDNameToHelmPath := map[string][]string{
-					"blackduck-postgres":       {"postgres", "podSecurityContext"},
-					"blackduck-init":           {"init", "securityContext"},
-					"blackduck-authentication": {"authentication", "podSecurityContext"},
-					"blackduck-binnaryscanner": {"binaryscanner", "podSecurityContext"},
-					"blackduck-cfssl":          {"cfssl", "podSecurityContext"},
-					"blackduck-documentation":  {"documentation", "podSecurityContext"},
-					"blackduck-jobrunner":      {"jobrunner", "podSecurityContext"},
-					"blackduck-rabbitmq":       {"rabbitmq", "podSecurityContext"},
-					"blackduck-registration":   {"registration", "podSecurityContext"},
-					"blackduck-scan":           {"scan", "podSecurityContext"},
-					"blackduck-uploadcache":    {"uploadcache", "podSecurityContext"},
-					"blackduck-webapp":         {"webapp", "podSecurityContext"},
-					"blackduck-logstash":       {"logstash", "securityContext"},
-					"blackduck-nginx":          {"webserver", "podSecurityContext"},
-					"appcheck-worker":          {"binaryscanner", "podSecurityContext"},
-					"blackduck-redis":          {"redis", "podSecurityContext"},
-					"blackduck-bomengine":      {"bomengine", "podSecurityContext"},
-					"blackduck-matchengine":    {"matchengine", "podSecurityContext"},
-					"blackduck-webui":          {"webui", "podSecurityContext"},
+					"blackduck-postgres":          {"postgres", "podSecurityContext"},
+					"blackduck-postgres-upgrader": {"postgresUpgrader", "podSecurityContext"},
```

Comment:
postgres-waiter should set to securityContext?

Reply:
I've added it.